### PR TITLE
pet life system framework

### PR DESCRIPTION
**Pull request by DeMuu**
Started building the base blocks of a „Pet Life System“. You can get
new pets and feed them when they’re hungry. 😋  🐈 🐕 🐟 🐹 

Samples in HTML file included

### CODE REVIEW

**Comment by DeMuu** [comment]
As it said "make it do something without breaking stuff" I thought a bit more than console logging would be good xD

**Comment by lynnaloo** [comment]
😹 This is great! I pulled in some other commits after you did the PR so there's some resolving to do. I'm going to put a couple code comments in too.

File: index.js [hunk 1]
```diff
@@ -1,4 +1,91 @@
-//cat's are always hungry
-var now = new Date();
-// the cat says
-console.log('Feed the kitties at ' + now);
+window.oranjs = window.oranjs || {};
+
+oranjs.Globals = {
```
Comment (lynnaloo) [inline index.js:3]: I don't know how comfortable you are with ES6, but this would be a perfect place to use classes versus prototypical inheritance.

Comment (lynnaloo) [reply index.js:3]: I'm OK with pulling it in as is for the #hacktoberfest fun, but just suggestion for future refactoring.

Comment (DeMuu) [reply index.js:3]: I'm familiar with ES6 but not familiar enough with classes so I thought it's better to get in the base functionality first. 😅 
Even though it's just a fun project, expanding the feature set a bit would be really fun IMO :)
Something like "fake pet AI". Sort of Tamagochi on the web

Comment (lynnaloo) [reply index.js:3]: Checkout my repo `fireball`. It's a Tamagotchi app that I used to teach JavaScript to teenagers and I have some issues in there as well.

**Comment by lynnaloo** [comment]
This evolved a bit into an Express web app over the last day. Do you mind merging some of your functionality into `app/app.js`? I'm going to merge some unit test changes, but this is the next "cool" fix to merge. As far as the pet AI, I would really love for you to contribute to https://github.com/lynnaloo/fireball!

**Comment by lynnaloo** [comment]
Also, fyi: https://github.com/lynnaloo/oranjs/issues/42

**Comment by DeMuu** [comment]
I'll be doing that soon (have been a bit busy at work recently). The Fireball project is quite a nice idea :)
I think leaving it simple is the main focus there but here we could maybe build more complex stuff basing it on an ES6 approach?

**Pull request closed without merging**